RELEASE.md

```diff
@@ -12,51 +12,109 @@ This page contains the release notes for AMD ROCm Software.
 
 -------------------
 
-## ROCm 6.0.1
+## ROCm 6.0.2
 
+In addition to library updates for several ROCm projects, this release includes bug fixes and
+improvements for the following libraries: hipBLAS, hipBLASLt, hipSPARSELt, rocAL, and rocBLAS. You
+can find additional details in the following section.
 
-### Library changes in ROCM 6.0.1
+### Library changes in ROCM 6.0.2
 
 | Library | Version |
 |---------|---------|
-| AMDMIGraphX | [2.8](https://github.com/ROCm/AMDMIGraphX/releases/tag/rocm-6.0.1) |
-| hipBLAS | [2.0.0](https://github.com/ROCm/hipBLAS/releases/tag/rocm-6.0.1) |
-| hipCUB | [3.0.0](https://github.com/ROCm/hipCUB/releases/tag/rocm-6.0.1) |
-| hipFFT | [1.0.13](https://github.com/ROCm/hipFFT/releases/tag/rocm-6.0.1) |
-| hipRAND | [2.10.17](https://github.com/ROCm/hipRAND/releases/tag/rocm-6.0.1) |
-| hipSOLVER | [2.0.0](https://github.com/ROCm/hipSOLVER/releases/tag/rocm-6.0.1) |
-| hipSPARSE | [3.0.0](https://github.com/ROCm/hipSPARSE/releases/tag/rocm-6.0.1) |
-| hipSPARSELt |  ⇒ [0.1.0](https://github.com/ROCm/hipSPARSELt/releases/tag/rocm-6.0.1) |
-| hipTensor | [1.1.0](https://github.com/ROCm/hipTensor/releases/tag/rocm-6.0.1) |
-| MIOpen | [2.19.0](https://github.com/ROCm/MIOpen/releases/tag/rocm-6.0.1) |
-| rccl | [2.15.5](https://github.com/ROCm/rccl/releases/tag/rocm-6.0.1) |
-| rocALUTION | [3.0.3](https://github.com/ROCm/rocALUTION/releases/tag/rocm-6.0.1) |
-| rocBLAS | [4.0.0](https://github.com/ROCm/rocBLAS/releases/tag/rocm-6.0.1) |
-| rocFFT | [1.0.25](https://github.com/ROCm/rocFFT/releases/tag/rocm-6.0.1) |
-| rocm-cmake | [0.11.0](https://github.com/ROCm/rocm-cmake/releases/tag/rocm-6.0.1) |
-| rocPRIM | [3.0.0](https://github.com/ROCm/rocPRIM/releases/tag/rocm-6.0.1) |
-| rocRAND | [2.10.17](https://github.com/ROCm/rocRAND/releases/tag/rocm-6.0.1) |
-| rocSOLVER | [3.24.0](https://github.com/ROCm/rocSOLVER/releases/tag/rocm-6.0.1) |
-| rocSPARSE | [3.0.2](https://github.com/ROCm/rocSPARSE/releases/tag/rocm-6.0.1) |
-| rocThrust | [3.0.0](https://github.com/ROCm/rocThrust/releases/tag/rocm-6.0.1) |
-| rocWMMA | [1.3.0](https://github.com/ROCm/rocWMMA/releases/tag/rocm-6.0.1) |
-| Tensile | [4.39.0](https://github.com/ROCm/Tensile/releases/tag/rocm-6.0.1) |
+| AMDMIGraphX | [2.8](https://github.com/ROCm/AMDMIGraphX/releases/tag/rocm-6.0.2) |
+| hipBLAS | [2.0.0](https://github.com/ROCm/hipBLAS/releases/tag/rocm-6.0.2) |
```

Comment:
Agree with Torre's comment. This link is dead, I don't see any 6.0.2 tag for hipBLAS, so not sure if the changes mentioned here will actually be in 6.0.2 or not. Likely no changes for hipBLAS 6.0.2 but can review again if the tag is made.

Reply:
okay, I'll remove hipBLAS from this release then.

Reply:
Shouldn't be removing hipBLAS from the release, the wrong release information is being collected.  Also wrong for rocBLAS.  @samjwu what branch is this information being gathered from?

Reply:
@TorreZuk Can you have another look? Sam reran his script for me, so hopefully I have the correct info now.

Reply:
@daineAMD do you mind also checking the updated file? I want to be super sure I'm collecting the proper info.

Reply:
Yes it looks good for me now, thanks!